### PR TITLE
text improvements #2020

### LINKED ISSUE #2020: Text improvements
There's some room for improvements in docs / log output.

### PR DESCRIPTION
**Pull request by sven-dmlr**
- typos/texts improved
 - backticks scramble up the generated HTML (accents)
 - docs: put examples as last information

closes #2020

### CODE REVIEW

File: sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java [hunk 1]
```diff
@@ -94,24 +94,23 @@ public enum PDSConfigDataKeyProvider implements PDSKeyProvider<ExecutionPDSKey>
      * Contains product timeout information
      */
     PDS_CONFIG_TIMEOUT_PRODUCT_MINUTES(new ExecutionPDSKey(PDSDefaultParameterKeyConstants.PARAM_KEY_PDS_CONFIG_PRODUCT_TIMEOUT_MINUTES,
-            "Maximum allowed time in minutes, before a producct will be timeout - means launcher script is automatically canceled by PDS").markSendToPDS()),
+            "Maximum allowed time in minutes, before a product will time out - means launcher script is automatically canceled by PDS").markSendToPDS()),
```
Comment (haerter-tss) [inline sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java:97]:
```suggestion
            "Maximum allowed time in minutes, before a product will time out - this means that the launcher script is automatically canceled by PDS").markSendToPDS()),
```
```suggestion
            "Maximum allowed time in minutes, before a product will time out - means launcher script is automatically canceled by PDS").markSendToPDS()),
```
```suggestion
            "Maximum allowed time in minutes, before a product will time out - means launcher script is automatically canceled by PDS").markSendToPDS()),
```
```suggestion
            "Maximum allowed time in minutes, before a product will time out - this means that the launcher script is automatically canceled by PDS").markSendToPDS()),
```

File: sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java [hunk 2]
```diff
@@ -38,7 +38,7 @@ public enum PDSConfigDataKeyProvider implements PDSKeyProvider<ExecutionPDSKey>
     PDS_CONFIG_USE_SECHUB_STORAGE(new ExecutionPDSKey(PDSDefaultParameterKeyConstants.PARAM_KEY_PDS_CONFIG_USE_SECHUB_STORAGE,
             "When 'true' the SecHub storage will be reused by PDS server. In this case SecHub will not upload job data to PDS.\n"
                     + "But it's crucial to have same root storage setup on PDS server side (e.g. same s3 bucket for S3 storage, or same NFS base for shared volumes).\n"
```
Comment (haerter-tss) [inline sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java:40]:
```suggestion
                    + "But it's crucial to have the same root storage setup on the PDS server side (e.g. same s3 bucket for S3 storage, or same NFS base for shared volumes).\n"
```

File: sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java [hunk 3]
```diff
@@ -94,24 +94,23 @@ public enum PDSConfigDataKeyProvider implements PDSKeyProvider<ExecutionPDSKey>
...
-            "This contains a comma separated list of path patterns for file excludes. These patterns can contain wildcards. Matching will be done case insensitive!"
-                    + "For example: `*.go,*.html, test1.txt` would excluded every go file, every HTML file and files named `test1.txt`.\n\n"
-                    + "When empty none of the files will be excluded. The exclude operation will be done AFTER the include file filtering has happend.")
-                            .markSendToPDS()),
+            "This contains a comma separated list of path patterns for file excludes. These patterns can contain wildcards. Matching will be done case insensitive!\n"
+                    + "When empty, then nothing will be excluded. The exclude operation will be done AFTER the include file filtering.\n\n"
+                    + "For example: '*.go,*.html, test1.txt' would exclude every Go file, every HTML file and files named 'test1.txt'.").markSendToPDS()),
 
     /**
      * This is automatically given to PDS by SecHub - depending on scan type. E.g.
```
Comment (haerter-tss) [inline sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java:108]:
```suggestion
     * This is automatically given to PDS by SecHub - depending on the scan type. E.g.
```

File: sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java [hunk 4]
```diff
@@ -94,24 +94,23 @@ public enum PDSConfigDataKeyProvider implements PDSKeyProvider<ExecutionPDSKey>
...
      * for a webscan this will be used to identify the current webscan target URL to
      * start scanning.
      */
     PDS_SCAN_TARGET_URL(new ExecutionPDSKey(PDSDefaultParameterKeyConstants.PARAM_KEY_PDS_SCAN_TARGET_URL,
-            "This contains the Target URL for current scan i.e webscan. Will not be set in all scan types. E.g. for a code scan this environment variable will not be available")
+            "This contains the target URL for current scan i.e webscan. Will not be set in all scan types. E.g. for a code scan this environment variable will not be available")
```
Comment (haerter-tss) [inline sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java:113]:
```suggestion
            "This contains the target URL for the current scan (i.e. webscan). Will not be set in all scan types. E.g. for a code scan this environment variable will not be available")
```

File: sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java [hunk 5]
```diff
@@ -38,7 +38,7 @@ public enum PDSConfigDataKeyProvider implements PDSKeyProvider<ExecutionPDSKey>
     PDS_CONFIG_USE_SECHUB_STORAGE(new ExecutionPDSKey(PDSDefaultParameterKeyConstants.PARAM_KEY_PDS_CONFIG_USE_SECHUB_STORAGE,
             "When 'true' the SecHub storage will be reused by PDS server. In this case SecHub will not upload job data to PDS.\n"
```
Comment (haerter-tss) [inline sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java:39]:
```suggestion
            "When 'true' the SecHub storage will be reused by the PDS server. In this case SecHub will not upload job data to PDS.\n"
```

File: sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java (hunk 2 above)
Comment (haerter-tss) [inline sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java:40]:
```suggestion
                    + "But it's crucial to have the same root storage setup on PDS server side (e.g. same s3 bucket for S3 storage, or same NFS base for shared volumes).\n"
```

**Review by haerter-tss — APPROVED** [review]
I added a couple of articles and fixed some typos.

File: sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java (hunk 2 above)
Comment (sven-dmlr) [reply sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java:40]: changed

File: sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java (hunk 3 above)
Comment (sven-dmlr) [reply sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java:108]: changed

File: sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java (hunk 4 above)
Comment (sven-dmlr) [reply sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java:113]: changed

File: sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java (hunk 5 above)
Comment (sven-dmlr) [reply sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java:39]: changed

File: sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java (hunk 2 above)
Comment (sven-dmlr) [reply sechub-commons-pds/src/main/java/com/mercedesbenz/sechub/commons/pds/PDSConfigDataKeyProvider.java:40]: same suggestion as #2 above

**Pull request merged**